app/views/additional_payments/claims/qualification.html.erb

```diff
@@ -1,58 +1,63 @@
-<% content_for(:page_title, page_title(t("additional_payments.questions.qualification.heading"), journey: current_journey_routing_name, show_error: current_claim.errors.any?)) %>
-<% path_for_form = current_claim.persisted? ? claim_path(current_journey_routing_name) : claims_path(current_journey_routing_name) %>
+<% content_for(:page_title, page_title(t("additional_payments.questions.qualification.heading"), journey: current_journey_routing_name, show_error: @form.errors.any?)) %>
```

Comment:
I just realised that Ken, Abi and I have been moving translation entries under `<journey>.forms.<slug>.questions`.

Reply:
👍